jsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+	"compilerOptions": {
+		"baseUrl": "./",
+		"paths": {
+			"@/*": ["src/*"],
```

Comment:
You sure this is necessary? It was introduced to Innopoints by Max and I never really saw the need in it, it's confusing

Reply:
It helps avoid writing a lot of '../../' imports. Also, this particular file does nothing but tell VS Code about that alias. The actual alias is defined in the rollup config

Reply:
But in this case it only substitutes `@/` for `src/`, why couldn't we just write `src/` directly?